inji-web/src/__tests__/components/Common/LandingPageWrapper.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { LandingPageWrapper, LandingPageWrapperProps } from '../../../components/Common/LandingPageWrapper';
+
+// Mock the i18n configuration
+jest.mock('react-i18next', () => ({
+    useTranslation: () => ({
+        t: (key: string) => key,
+    }),
+}));
+
+const renderWithRouter = (props: LandingPageWrapperProps) => {
```

Comment:
move this methods to mockUtils.tsx

Reply:
The function renderwithprovider has been moved to mockutils , but in some files we have declared them locally as we are facing some problems when referring that from mockutils